plugins/grafana-custom-plugins/grafana-dependency-plugin/README.md

```diff
@@ -114,7 +143,8 @@ installed panels will appear. For more information, visit the docs on [Grafana p
 
 Users can customize the panel by editing its options and choosing to group the
 diagram based on a chosen Pod label. It is also possible to change the color
-of the Pod squares in the diagram.
+of the Pod squares in the diagram. Additionally, users can switch betwen layer
+4 and layer 7 visualizations.
 
 <img src="https://user-images.githubusercontent.com/10016630/233191376-7cf471b8-5e3e-473a-8696-da25ab981066.png" width="400" alt="Panel Option Editor View">
 
```

Comment:
I think we need to update the section 4 as you already migrate the dependency-plugin from grafana/toolkit to create-plugin?